specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/virtualWan.json

```diff
@@ -5724,6 +5803,62 @@
           "description": "The ip address of the next hop."
         }
       }
+    },
+    "EffectiveRouteList": {
```

Comment:
Is this definition used?

Reply:
Added this definition so that we can use it in Powershell when we expose the corresponding cmdlet